lib/cdo/aws/cloudfront.rb

```diff
@@ -245,7 +245,9 @@ def self.config(app, reference = nil)
                 origin_ssl_protocols: {
                   quantity: 2,
                   items: %w(SSLv3 TLSv1)
-                }
+                },
+                origin_read_timeout: 5,
+                origin_keepalive_timeout: 30
```

Comment:
@joshlory turns out these values are mismatched, they should have been `origin_read_timeout: 30, origin_keepalive_timeout: 5` to match the defaults. sorry for not picking this up earlier